### ISSUE TITLE
[API] Configure role claim

### ISSUE THREAD

**Issue by thisIsTheFoxe** [skoruba/IdentityServer4.Admin#485]
Hi,

While implementing I noticed that `TokenValidationClaimRole` is only available in the Admin-UI Project.

The API authorization looks like [this](https://github.com/skoruba/IdentityServer4.Admin/blob/be6fc55a61d890378dd21f9110c2850e963a487f/src/Skoruba.IdentityServer4.Admin.Api/Helpers/StartupHelpers.cs#L240) where `JwtClaimTypes.Role` defaults to "role". In many cases, however, the actual role claim may be different..? <sub>(at least in my case..)</sub> 

Would be great if it was possible to also have something like `TokenValidationClaimRole` together with the already existing `AdministrationRole` in the API's `appsettings.json` to be able to configure admin-rights even better.. :)

**Comment by skoruba** [comment]
Hey, do you mean this one? - 
https://github.com/skoruba/IdentityServer4.Admin/blob/master/src/Skoruba.IdentityServer4.Admin.Api/appsettings.json#L16

**Comment by thisIsTheFoxe** [comment]
Exactly, but that only let’s you define the role-value.. the name (or key) of the role claim is `JwtClaimTypes.Role` which is the constant string "role". 

Then again if #478 is the current idea, you would probably just have **one** setting file where you can specify both the role-claim and the actual role-value.. just as in the Admin project. ^^
